### PR TITLE
remove hook to be helm3 compatible, change crd generation, move to crds directory

### PR DESCRIPTION
**Pull request by 21h**
crd hook not supported in helm3, tested installation without this hook and all works fine.

### CODE REVIEW

**Comment by 21h** [comment]
@AMecea checkout files, I made changes you requested

File: hack/generate_chart_manifests.sh [hunk 1]
```diff
@@ -3,22 +3,14 @@
 CHART_PATH=../charts/mysql-operator
 CONFIG_PATH=../config
 
-# templates/crds.yaml update
-awk 'FNR==1 && NR!=1 {print "---"}{print}' ${CONFIG_PATH}/crds/*.yaml > ${CHART_PATH}/templates/_crds.yaml
-yq m -d'*' -i ${CHART_PATH}/templates/_crds.yaml chart-metadata.yaml
-yq w -d'*' -i ${CHART_PATH}/templates/_crds.yaml 'metadata.annotations[helm.sh/hook]' crd-install
```
Comment (AMecea) [inline hack/generate_chart_manifests.sh]: the annotations `helm,sh/hook=crd-install` should be kept to be backward compatible.

File: charts/mysql-operator/crds/crds.yaml [hunk 2]
```diff
@@ -23,14 +16,8 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   labels:
-    app: '{{ template "mysql-operator.name" . }}'
```
Comment (AMecea) [inline charts/mysql-operator/crds/crds.yaml]: I suggest keeping this label to a hardcoded value: `app=mysql-operator`

**Comment by AMecea** [comment]
Also, please update the chart `README.md` and also bump chart version, see #445

**Comment by AMecea** [comment]
ping @21h  :smile:

**Comment by vitobotta** [comment]
Hi @21h and @AMecea , I have tried several times  with the changes shown in this PR, and I can install the operator with these changes, but backups do not work for some reason. If I install with Helm 2, backups work just fine. @21h Have you tested backups with your changes? Thanks!

**Comment by vitobotta** [comment]
I have a doubt.... I have only made the changes to the CRD file and moved it to the crds directory. Do I need to use the script generate_chart_manifests.sh as well? Thanks

**Comment by vitobotta** [comment]
In the logs for the failing backup jobs I see this:

```
sidecar take backup command failed      {"error": "getting backup: fail to get backup: Get http://mysql-cluster-mysql-0.mysql.mysql:8080/xbackup: dial tcp: i/o timeout, code: unknown"}
```

I tried to curl Is mysql-cluster-mysql-0.mysql.mysql:8080 from a Ubuntu container in the same namespace and I could connect. I wonder why the backup job cannot connect to it?

**Comment by vitobotta** [comment]
I found my problem.... Even though I checked out the 0.3.8 tag, the images in the values.yaml are set to "latest", instead of "0.3.8". Changing that fixes the backups.

**Comment by AMecea** [comment]
Closing this in favor of #468. In #468 I've cherry-picked your commits and squashed them and add the rest of changes. Please test it and let me know it solves your problem. Thank you for your contribution!

**Pull request closed without merging**